Do-It/Model/DoIts/ReadingDoIt.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  ReadingDoIt.swift
+//  Do-It
+//
+//  Created by Aaron on 1/16/19.
+//  Copyright © 2019 The Swifter Picker-Uppers. All rights reserved.
+//
+
+import Foundation
+
+struct ReadingDoIt : DoIt {
+    var id: DoItId
+    var course: Course
+    var dueDate: Date
+    var description: String
+    var name: String
+    var priority: DoItPriority
+    
+    let pageStart : Int
+    let pageEnd : Int
+    
+    
+    init(id: DoItId, course: Course, dueDate: Date, description: String,
```

Comment:
One of the big perks of using structs rather than classes is that Swift generates these initializers automatically for us—you should be able to remove these inits without a problem.

Reply:
That's so nice